### ISSUE TITLE
[NuGet.org Bug]: A11y_NuGet Service_.Json_Version History_Screenreader : Narrator is not announcing column name as column header for "Version and Downloads".

### ISSUE THREAD

**Issue by joelverhagen** [NuGet/NuGetGallery#8922]
### Impact

It's more difficult to complete my work

### Describe the bug

Source: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1438720

### Repro Steps

1. Hit the URL: https://www.nuget.org/ 
1.  Tab navigate to "Sign in" control and hit enter.
1.  Tab navigate to "Sign in with Microsoft" control and hit enter.
1.  Search for a key term (say “json”) from search box.
1.  Click on "Newtonsoft.json" control and hit enter.
1.  Tab navigate to "Version History" control and hit enter.
1.  Turn on Narrator
1. Check Narrator is announcing column name as column header for "Version and Downloads".

Narrator is not announcing column name as column header for "Version and Downloads" column.

Note:
No issue observed in NVDA 
This issue is also reproducing in Statistics page also.

### Expected Behavior

Narrator should be announce column name as column header for "Version and Downloads" column.

**Comment by shishirx34** [comment]
external bug

**Comment by dannyjdev** [comment]
Resolved the AzDo item and assigned it to the tester. Explanation of why I consider this to be an external bug is on the AzDo item.
